app/policies/case/sar_policy.rb

```diff
@@ -89,6 +89,11 @@ def can_request_further_clearance?
     !self.case.flagged?
   end
 
+  # TODO (Mohammed Seedat): Replace with rules for authorization
+  def extend_deadline_for_sar?
+    true
```

Comment:
Personally I like to have this logic in the state machine so here we would have 
```
clear_failed_checks
    check_can_trigger_event(:extend_deadline_for_sar)
```
which will check the permissions set in the state machine